### DISCUSSION TITLE
Compiler error

### DISCUSSION THREAD

**Discussion by TheRealDood** [KadeArchive/Kade-Engine#359] [Q&A] [answered]
Trying to compile the game then i get these errors
![Capture](https://user-images.githubusercontent.com/83540641/116832081-27f13600-ab68-11eb-82bf-07719fd82863.PNG)
kinda stumped and have no idea what to do. (new to this kind of stuff)
i did edit some things but still have no idea whats wrong, even tried the original unedited file, and i still get these errors

**Comment by puyoxyz** [comment, marked as answer]
Can you send the output of `haxe --version` and `haxelib list` please?

**Reply by TheRealDood** [reply]
sure

**Reply by TheRealDood** [reply]
![image](https://user-images.githubusercontent.com/83540641/116832283-43107580-ab69-11eb-92f3-478dadf7306b.png)

**Reply by puyoxyz** [reply]
Can you try using Haxe `4.1.5`?

**Reply by TheRealDood** [reply]
sure, let me just install it real quick

**Reply by TheRealDood** [reply]
looks like that was the problem, sorry to bother ya man. have a good day!
